install_redhat.md

```diff
@@ -71,18 +71,18 @@ If not, install GraphicsMagick:
 Configure Apache
 ----------------
 
-    sudo cp /opt/chronam/conf/chronam.conf /etc/httpd/conf.d/chronam.conf
-    sudo install -o `whoami` -g users -d /opt/chronam/static
-    sudo install -o `whoami` -g users -d /opt/chronam/.python-eggs
+    sudo cp /opt/openoni/conf/openoni.conf /etc/httpd/conf.d/openoni.conf
+    sudo install -o `whoami` -g users -d /opt/openoni/static
+    sudo install -o `whoami` -g users -d /opt/openoni/.python-eggs
 
 Update the KeepAlive directive in /etc/httpd/conf/httpd.conf config from 'Off' 
 to 'On'. If you are the Library of Congress you will also want to canonicalize 
 URLs that used by the Chronicling America application at the Library of Congress:
 
-    sudo cp /opt/chronam/conf/chronam-canonical.conf /etc/httpd/conf.d/
+    sudo cp /opt/openoni/conf/openoni-canonical.conf /etc/httpd/conf.d/
 
 
 Continue
 --------
 
-* You can now return to the Install section in [README.md](https://github.com/LibraryOfCongress/chronam/blob/master/README.md#install)
+* You can now return to the Install section in [README.md](https://github.com/LibraryOfCongress/openoni/blob/master/README.md#install)
```

Comment:
This needs to be a link to our repo - there's a cool way to link from markdown files to other files in the same repository if you feel like figuring that out, but I don't recall it offhand.